Add license (#2)
pyinstaller/pyinstaller-hooks-contrib: LICENSE.APL.txt

==========================================================
Licensing Terms for he PyInstaller community runtime hooks
==========================================================

> The PyInstaller community **runtime** hooks are licensed under the terms of the

Reviewer: Again: Maybe add something like "these reside in rthooks/"?

Author: Done.

Apache 2.0 Software License as published by the Apache Software Foundation.

> Please note that this license only applies to files which have the following header:

Reviewer: ```suggestion
Please note that this license only applies to files which state it in the header, like this:
```

Author: Done.

# ------------------------------------------------------------------
# Copyright (c) 2020 PyInstaller Development Team.
#
# This file is distributed under the terms of the Apache License 2.0
#
# The full license is available in LICENSE.APL.txt, distributed with
# this software.
#
# SPDX-License-Identifier: Apache-2.0
# ------------------------------------------------------------------

Apache License 2.0

> ++++++++++++++++++

Reviewer: I'm curious: While you added the complete text of the GPL, but not of the APL. Any reason?

Author: https://github.com/pyinstaller/pyinstaller-hooks-contrib/pull/2#discussion_r408765869

>    Copyright 2020 PyInstaller Development Team

Reviewer: Huh? This reads like the Apache License 2.0 (see headline above) is Copyrighted by us. Is this intended?

Author: https://choosealicense.com/licenses/apache-2.0/ At the bottom of the license:

```
END OF TERMS AND CONDITIONS

   APPENDIX: How to apply the Apache License to your work.

      To apply the Apache License to your work, attach the following
      boilerplate notice, with the fields enclosed by brackets "[]"
      replaced with your own identifying information. (Don't include
      the brackets!)  The text should be enclosed in the appropriate
      comment syntax for the file format. We also recommend that a
      file or class name and description of purpose be included on the
      same "printed page" as the copyright notice for easier
      identification within third-party archives.

   Copyright [yyyy] [name of copyright owner]

   Licensed under the Apache License, Version 2.0 (the "License");
   you may not use this file except in compliance with the License.
   You may obtain a copy of the License at

       http://www.apache.org/licenses/LICENSE-2.0

   Unless required by applicable law or agreed to in writing, software
   distributed under the License is distributed on an "AS IS" BASIS,
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.`
```

Reviewer: Hmm, re-reading this, I would understand this is the boilerplate to be put into *each* file:
> attach the following boilerplate notice […] The text should be enclosed in the appropriate
      comment syntax for the file format

E.g. in https://github.com/apache/freemarker/ this is in each file (I checked a single one, though ;-) and there is a "LICENCE" file containing the license text.
